README.md

```diff
@@ -1,61 +1,64 @@
+This facet extends upon Aragon's token voting. Instead of wallets only being able to vote once with all their voting power, proposals can be configured to allow people to vote multiple times with a part of their voting power. This plugin also supports multiple proposal types and a cap on voting power for any single wallet. Partial voting is based on the idea that people do not have super strong opinions on all proposals and thus do not want to influence the vote massively.
 #### Partial Burn Voting
-This facet is partial voting with the addition of all used voting power being burned. If the proposal voted on does not reach the participation threshold, the plugin can be configured for these tokens to be refundable. Partial burn voting is a way to make sure people will only vote on proposal that they care about with an amount of voting power based on how much they care.
+This facet is partial voting, with the addition of all used voting power being burned. If the proposal voted on does not reach the participation threshold, the plugin can be configured to make the burned tokens refundable. Partial burn voting is a way to make sure people will only vote on proposals that they care about, with an amount of voting power based on how much they care.
 #### Partial Burn Voting Proposals
-This facet adds to the proposal creation of partial voting. Proposal creation will burn an amount of voting power of the proposer. The plugin can be configured for these tokens to be refundable if the created proposal passes. Partial burn voting proposals is created to incentivize people to put effort and thought into their proposal and thus filters out ridiculous and low quality proposals.
+This facet adds to the proposal creation of partial voting. Proposal creation will burn an amount of voting power of the proposer. The plugin can be configured to make the burned tokens refundable if the created proposal passes. Partial burn voting proposals is created to incentivize people to put effort and thought into their proposals and thus filter out ridiculous and low quality proposals.
 ### Structures
-Multiple governance structures have been defined for plugins to interact with each other. These structures allow future facets to be compatible with all existing plugins using the implementented structure interface.
+Multiple governance structures have been defined for plugins to interact with each other. These structures allow future facets to be compatible with all existing plugins using the implemented structure interface.
 #### Membership
-These strucutres are related to what wallets are considered to be members of the DAO and other member related information, such as ranks.
+These structures are related to what wallets are considered members of the DAO and other member-related information, such as ranks.
 ##### Base
 All these plugins are based on the IMembership of Aragon.

```

Comment:
I would make the "IMembership" a code thingie, so `IMemership`, with a link to wherever this interface is defined.
I.e., something like this:
"All these plugins are based on the [``IMembership``](https://github.com/aragon/...?) interface from Aragon."